### PR TITLE
5-get-review-id

### PR DESCRIPTION
**Pull request by reawilliams20**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -66,4 +66,45 @@ describe('/api/reviews', () => {
...
+});
+
+describe('/api/reviews/:review_id', () => {
+    test('200, responds relevant review', () => {
+      const review_id = 2;
+      return request(app)
+        .get(`/api/reviews/${review_id}`)
+        .expect(200)
+        .then(({body: {review}}) => {
+            expect(review).toBeInstanceOf(Object);
```
Comment (RichardNeat) [inline __tests__/app.test.js]: We don't need to explicitly check for an object here as we are matching it in the next expect block

File: app.js [hunk 2]
```diff
@@ -2,15 +2,37 @@ const express = require("express");
...
-app.use(express.json());
+
 
 app.get('/api/categories', getCategories);
 app.get('/api/reviews', getReviews);
+app.get('/api/reviews/:review_id', getReview);
+
 app.all('*', handle404Routes);
 
+//handles sql query error
```
Comment (RichardNeat) [inline app.js:18]: Nice error handlers, we could extract these into their own 'error-handling.js' file.

**Review by RichardNeat — COMMENTED** [review]
Nice work! Just a couple of minor comments if you could please make those changes before merging, once that's done and committed go ahead and merge :)

File: app.js (hunk 2 above)
Comment (reawilliams20) [reply app.js:18]: Hi Richard, I'm having trouble with extracting these to a separate file. It doesn't work

**Pull request merged**